### ISSUE TITLE
A11y_AzureDataStudio_New deployment: SQL Server 2019 Big Data Cluster_Screen Reader: Narrator is not announcing the cells with column header..

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18567]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info & Relationships; #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on  the Narrator. 
2.Launch Azure Data Studio.
3.Select the . Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment".
5. In the Deployment options, select "SQL Server Big Data Cluster".
6. In Step-1 of Deployment pre-requisites , there is a Required tools table.
7.Verify whether that the narrator is announcing the cells with column header or not.

### Note:
Similar issue can be observed in throughout the application whether the tables are present.

### Actual Result:
Narrator is not announcing the cells with column header.

### Expected Result:
Narrator should announced the cells with column header.

### Attachment:
**Narrator attachment:**
![A11y_AzureDataStudio_New deployment SQL server big cluster_Narrator](https://user-images.githubusercontent.com/99962579/155976749-592e770f-210b-4d95-b5f0-50b219414afe.gif)
**NVDA Attachment:**
**![A11y_AzureDataStudio_New deployment SQL server 2019_NVDA](https://user-images.githubusercontent.com/99962579/155981396-5705f2cb-5bff-4665-a705-e80314c38034.gif)**

**Comment by Charles-Gagnon** [comment]
Verified this isn't working for Narrator, but it is working for NVDA

`kubectl  Tool  column 1`

So closing this as it seems to be an issue with Narrator, we have the correct `columnheader` role applied to the header element so it should be recognizing that.

**Comment by Gayathri1501** [comment]
Verified in this environment by the guidelines of Narrator team as the shortcut using the Ctrl+Shift+Alt+Up arrow, the cells with column header are announcing. Hence, the issue is fixed. Plese find the below attachment.
![A11y_AzureDataStudio_New deployment SQL Server 2019 Big Data Cluster_Screen Reader_](https://user-images.githubusercontent.com/99962579/178902474-8b6142da-b825-4d3a-a1bd-718f5bd410fc.gif)

**Comment by Samhitha66** [comment]
#Closed;
